src/main/java/com/configcat/LogFilterFunction.java

```diff
@@ -0,0 +1,15 @@
+package com.configcat;
+
+@FunctionalInterface
+interface LogFilterFunction {
+
+    /**
+     *
+     * @param eventId
+     * @param logLevel
+     * @param message
+     * @param exception
+     * @return
+     */
+     boolean apply(int eventId, LogLevel logLevel , String message, Throwable exception);
```

Comment:
One more last bit of hair-splitting: other SDKs use the parameter order `logLevel`, `eventId`, `message`, `exception` in their public interfaces. E.g.:
* https://github.com/configcat/.net-sdk/blob/v9.2.0/src/ConfigCatClient/Logging/IConfigCatLogger.cs#L22
* https://github.com/configcat/common-js/blob/v9.3.0/src/ConfigCatLogger.ts#L77

It's not that this order is better or something, I'd just like to keep it consistent as much as possible because consistency improves maintainability and also helps users with onboarding and switching between SDKs.